github_viewer_slack/post.py

```diff
@@ -9,27 +9,41 @@
 import json
 
 
-def post(message):
+def post(**blob):
+    query = {
+        'as_user':      True,
+        'text':         None,
+        'username':     None,
+        'parse':        None,
+        'link_names':   None,
+        'attachments':  None,
+        'unfurl_links': None,
+        'unfurl_media': None,
+        'icon_url':     None,
+        'icon_emoji':   None
+    }
+    query.update(blob)
     slacker = Slacker(API_TOKEN)
-    slacker.chat.post_message(Channel, message, as_user=True)
+    slacker.chat.post_message(Channel, **query)
 
 
 def post_by_repo(user_name, repo_name):
     commits = list(get_post_commits(user_name, repo_name))
     if len(commits) == 0:
         return
 
-    last_commit_sha = commits[0]['sha']
-    text = "{}/{} is comitted: \n".format(user_name, repo_name)
+    attachment_list = []
+    text = "%s/%s is comitted\n" % (user_name, repo_name)
```

Comment:
`2.`  
As a result of I seeing, I think this is better 🤔   
Would you like examine again ?

`1.`  
OK!
I'll fix it :+1: